mne/viz/_3d.py

```diff
@@ -1344,3 +1346,160 @@ def _get_view_to_display_matrix(scene):
                                  [0.,            0.,   1.,        0.],
                                  [0.,            0.,   0.,        1.]])
     return view_to_disp_mat
+
+
+def plot_dipole_mri_orthoview(dipole, trans, subject, subjects_dir=None,
```

Comment:
also how do you specify what dipole you plot? it seems you only plot the first one.